standup/status/models.py

```diff
@@ -12,7 +12,13 @@ class Project(models.Model):
     name = models.CharField(max_length=100, unique=True)
     slug = models.CharField(max_length=100, unique=True)
     color = models.CharField(max_length=6)
-    repo_url = models.URLField()
+    repo_url = models.URLField(max_length=100)
+
+    class Meta:
+        db_table = 'project'
```

Comment:
The current names are pretty reasonable and should not conflict with future names that Django will use. I'm thinking we just leave it, but I could be convinced otherwise. I'm pretty sure I could write a migration that would pretty easily just rename the tables; I'm just not sure there's much point.